apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -734,21 +804,25 @@ def grade(element_html, data):
                         feedback += FIRST_WRONG_FEEDBACK["block-group"]
                     feedback += "</ul>"
 
-    data["partial_scores"][answer_name] = {
+    data["partial_scores"][answer_name] = {  # type: ignore
         "score": round(final_score, 2),
         "feedback": feedback,
         "weight": answer_weight,
         "first_wrong": first_wrong,
     }
```

Comment:
@eliotwrobson 

I'm not sure what the best things to do is, typechecking fails because of the `first_wrong` key we have here, which isn't declared as part of the `pl.PartialScore` type. I don't really want to add it on there because its not something that other elements will have. 

It would be cool if elements could define their own `PartialScore` type and then pass it as a type parameter to `pl.QuestionData`. It looks like as of python 3.11, classes that inherit from TypedDict can take type parameters (https://github.com/python/cpython/issues/89026), so we could probably make this work. (as an aside, we could also use that new feature to get rid of a bunch of the `Any`s in `pl.QuestionData`!)

Reply:
Wait a minute, on close inspection, it looks like the `"first_wrong"` thing here is actually never being used. I think we can just remove it.

Reply:
(this could potentially break someone's question if they access it in their `server.py` file for some reason, but that seems very very unlikely)

Reply:
It looks like you removed this in the latest commit?

Reply:
Ya, I just went ahead and removed it. I think the chance of it breaking someones question is vanishingly small